images/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:stable as java8-builder
+
+RUN apt update -qq &&\
+    apt install wget -y
+
+RUN wget https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz -P /srv && \
```

Comment:
`/srv`? why not java?

Reply:
this is multi-stage docker, in the final docker image it is `/usr/share/java`
in this stage, its only purpose is to "serve" java in this directory for the later stage